CHANGELOG.md

```diff
@@ -5,17 +5,42 @@ All notable changes to Zebra are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org).
 
-## [Zebra 1.0.0-rc.6](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-rc.6) - 2023-03-TODO INSERT DATE HERE
+## [Zebra 1.0.0-rc.6](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-rc.6) - 2023-03-23
 
-This release:
-- TODO: insert summary here
+This release we fixed several security issues.
 
 ### Known Issues
 
-- `orchard` 0.3.0 can't verify `halo2` proofs when compiled with Rust 1.69 or later (currently nightly Rust TODO: update this based on https://releases.rs/ ). Compile Zebra with stable Rust to avoid this bug. ([halo2/#737](https://github.com/zcash/halo2/issues/737))
-    - TODO: turn this into a breaking change if the bug is fixed in `orchard` 0.3.1 or 0.4.0, and [Zebra has updated #6232](https://github.com/ZcashFoundation/zebra/issues/6232)
+- `orchard` 0.3.0 can't verify `halo2` proofs when compiled with Rust 1.69 or later (currently beta Rust). Compile Zebra with stable Rust to avoid this bug. ([halo2/#737](https://github.com/zcash/halo2/issues/737)).
+Zebra tracking issue for this problem is [here](https://github.com/ZcashFoundation/zebra/issues/6232).
+
+### Security
+
+- Fix Halborn disclosures in Zebra ([#6297](https://github.com/ZcashFoundation/zebra/pull/6297))
```

Comment:
Do you think that the Halborn disclosures should be in the release summary?
We could use some of the descriptions from the blog post, and also link to it so people can find more information:
https://zfnd.org/statement-on-recent-security-disclosures-by-halborn/

Reply:
Added to the summary at https://github.com/ZcashFoundation/zebra/pull/6314/commits/845dea26ca81ad7429990446b5232fa9b0dbdb68